test/tests_for_sdkmr/ee2_SDKMethodRunner_test.py

```diff
@@ -921,6 +926,17 @@ def test_check_jobs_date_range(self, condor_mock):
         tomorrow = now + timedelta(days=1)
         day_after = now + timedelta(days=2)
 
+        print(
+            f"Last month - 1 hour: {last_month_and_1_hour} "
+            + f"ts: {last_month_and_1_hour.timestamp()}"
+        )
+        print(f"Last month:          {last_month} ts: {last_month.timestamp()}")
+        print(f"Last Week:           {last_week} ts: {last_week.timestamp()}")
+        print(f"Yesterday:           {yesterday} ts: {yesterday.timestamp()}")
+        print(f"Now:                 {now} ts: {now.timestamp()}")
+        print(f"Tomorrow:            {tomorrow} ts: {tomorrow.timestamp()}")
+        print(f"Day after:           {day_after} ts: {day_after.timestamp()}")
```

Comment:
I'm not a big fan of printing out a lot of stuff during tests, but it seems that the ee2 tests are rather fond of it. Hrm.

Reply:
FWIW, this PR reduces the amount of lines printed out since it prints out all the test times once at the start of the test instead of once per job in several loops.

Reply:
Also pytest dumps any printouts unless the test fails, so it doesn't clutter the results if the test passes